akka-docs/src/main/paradox/typed/persistence-snapshot.md

```diff
@@ -1,3 +1,6 @@
+---
+project.description: Append only event logs, snapshots and recovery with Akka event sourced actors.
+---
```

Comment:
Not really a landing page?

Reply:
I agree, though I thought snapshotting a common topic one would search for. WDYT?